test/integ/schema.test.ts

```diff
@@ -43,3 +43,56 @@ it("fauna schema push --stage --force works", async () => {
     )
   ).to.deep.equal(null);
 });
+
+const status = async (secret: string) => {
+  const output = await shellOk("fauna schema status --dir .", secret);
+
+  // Remove the "Connected to endpoint" line.
+  return output.split("\n").slice(1).join("\n");
+};
+
+it("fauna schema status works", async () => {
+  const secret = await newDB();
+
+  await shellOk(
+    "fauna schema push --dir test/integ/schema/start --force",
+    secret
+  );
+
+  expect(await status(secret)).to.equal(
+    stripMargin(
+      `|Status: none
+       |`
+    )
+  );
+
+  await shellOk(
+    "fauna schema push --dir test/integ/schema/staged_index --force --stage",
+    secret
+  );
+
+  expect(await status(secret)).to.equal(
+    stripMargin(
+      `|Status: ready
+       |The schema is ready to be committed.
+       |
+       |Staged changes:
+       |* Modifying collection \`User\` at 0:90/main.fsl:
+       |  * Summary:
+       |    + added: index \`byName\` (see diff)
+       |
+       |  * Diff:
+       |      collection User {
+       |        name: String
+       |        email: String
+       |    +
+       |    +   index byName {
+       |    +     terms [.name]
+       |    +   }
+       |      }
+       |
+       |
+       |`
```

Comment:
these are some interesting newlines...

Reply:
Yes indeed. I think two are from core, and one is from the shell.